indexer/ai/src/main/java/com/google/cloudsearch/ai/AISkill.java

```diff
@@ -0,0 +1,43 @@
+package com.google.cloudsearch.ai;
+
+
+import java.util.List;
+
+import com.google.common.collect.Multimap;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+
+/**
+    Interface for all Skills.
+ */
+public interface AISkill {
+
+    public void setAISkillName(String aiSkillName);
+    public String getAISkillName();
+
+    public void setOutputMappings(JSONArray outputMapping);
+    public List<OutputMapping> getOutputMappings();
+
+    public void setInputs(JSONObject input);
+    public JSONObject getInputs();
```

Comment:
Change the return Type to Map<String, String>? Also, is this getter being used currently?

Reply:
No, it is not used currently, just kept it for uniformity. We can remove this later.